JavaScript/JavaScriptSDK/AppInsights.ts

```diff
@@ -131,6 +133,7 @@ module Microsoft.ApplicationInsights {
 
             if (!this.config.disableAjaxTracking) {
```

Comment:
Thanks for nailing the unit tests @	MaximBalaganskiy . I missed adding a comment here, can you add a disableFetchTracking and default it to true for the fetchMonitor. We can take changes in PROD and test. Also, this can potentially impact customer bill, so we cannot start collecting by default.